WarOfEmpires.Domain/Attacks/TroopDefinition.cs

```diff
@@ -0,0 +1,22 @@
+﻿using WarOfEmpires.Domain.Common;
+using WarOfEmpires.Domain.Empires;
+
+namespace WarOfEmpires.Domain.Attacks {
+    public class TroopDefinition {
+        public TroopType Type { get; }
+        public BuildingType BuildingType { get; }
+        public int BaseAttack { get; }
+        public int BaseDefence { get; }
+        public Resources Cost { get; }
+        public string Name { get; }
```

Comment:
Is the Name property necessary? It looks identical to TroopType.Type.ToString(), and I can't see that it is referenced outside the TroopDefinition

Reply:
No it is not.